src/main/java/org/broadinstitute/dsm/model/rgp/AutomaticProbandDataCreator.java

```diff
@@ -116,7 +116,6 @@ private boolean extractAndInsertProbandFromESData(DDPInstance instance, ElasticS
                         new FamilyMemberDetails(firstName, lastName, memberType, familyId, collaboratorParticipantId);
                 probandMemberDetails.setMobilePhone(mobilePhone);
                 probandMemberDetails.setEmail(email);
-                probandMemberDetails.setApplicant(true);
```

Comment:
I think we're using `is_applicant` in `ParticipantUtil.matchesApplicantEmail()`?